src/lib/rt-categorize-html.js

```diff
@@ -116,24 +116,21 @@ export function rt_categorize_html() {
 
     // create response function
     var after_response = function(info) {
-
       // kill any remaining setTimeout handlers
       jsPsych.pluginAPI.clearAllTimeouts();
 
       // clear keyboard listener
       jsPsych.pluginAPI.cancelAllKeyboardResponses();
 
-      var correct = false;
-      if (trial.key_answer === info.key) {
-        correct = true;
-      }
+      const correct = trial.key_answer === info.key - 96 || trial.key_answer === info.key - 48;
 
       // save data
       trial_data = {
         "rt": info.rt,
         "correct": correct,
         "stimulus": trial.stimulus,
-        "key_press": info.key
+        "key_press": info.key,
+        "key_answer": trial.key_answer
```

Comment:
@mcmcgrath13 is this what you meant with the above comment?

Reply:
yup!